fidesctl/src/fidesctl/core/apply.py

```diff
@@ -109,39 +70,40 @@ def apply(
 ) -> None:
     """
     Apply the current manifest file state to the server.
-    Excludes systems and registries.
```

Comment:
This is certainly not a true statement after these changes. I don't believe this was a true statement before these changes either. If I'm incorrect about this, and there's a good reason to skip system and registry resources, please let me know.

Reply:
this is a super legacy comment, it is incorrect